composeApp/src/commonMain/kotlin/home.kt

```diff
@@ -64,11 +98,65 @@ fun HomeElement(homeData: HomeData?, onResetConfig: () -> Unit) {
             modifier = Modifier.fillMaxSize().padding(horizontal = 16.dp),
             verticalArrangement = Arrangement.Center
         ) {
+
+            var parsedResponse: Test? by remember { mutableStateOf(null) }
+            var errorMessage: String? by remember { mutableStateOf(null) }
+
+            val coroutineExceptionHandler = CoroutineExceptionHandler { _, throwable ->
+                throwable.printStackTrace()
+            }
+
+            openIdConnectClient?.let {
+                val httpClient = createClient(it, openId, tokenStore)
+
+                CoroutineScope(Dispatchers.IO + coroutineExceptionHandler).launch {
+                    val response = httpClient.post("/api/v1/test")
```

Comment:
A Test to validate that authentication works.
We probably should migrate to typesafe requests anyways.
https://ktor.io/docs/client-resources.html#resource_classes